utils/releaser/tests/FileManipulator/ChangelogFileManipulator/Source/CHANGELOG-after.md

```diff
@@ -0,0 +1,7 @@
+## v1.0.0 - %d-%d-%d
+
+Some content
+
+Some old content
+
+[v1.0.0]: https://github.com/shopsys/shopsys/compare/v0.9.0...v1.0.0
```

Comment:
`v` in [] is wrong but `v` in link is OK, now i see that it is strange so please, do not hesitate to contact @vitek-rostislav

Reply:
let us discuss it...